### ISSUE TITLE
five.async usage in browser

### ISSUE THREAD

**Issue by nikeee** [jackdcrawford/five#197]
The current implementation of five.async uses process.nextTick, which is a nodejs specific API. This won't work in a browser apparently.
I'd fix this by checking if there is a process variable. I could make a PR soon.

**Comment by shackpank** [comment]
If you need that in the browser, and it's not currently possible, feel free to put a PR in!

**Comment by nikeee** [comment]
Indeed, it's pretty essential for the scalability of my application.

**Comment by mbixby** [comment]
This method could return a promise.

**Comment by nikeee** [comment]
I think we can now assume a fair amount of promise support. PR still welcomed?
